dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -1084,6 +1085,8 @@ BlockInputStreams DeltaMergeStore::read(const Context & db_context,
                 extra_table_id_index,
                 physical_table_id,
                 log_tracing_id);
+            auto * unordered_stream = dynamic_cast<UnorderedInputStream *>(stream.get());
+            unordered_stream->setRuntimeFilterInfo(runtime_filter_list, rf_max_wait_time_ms);
```

Comment:
maybe can add `runtime_filter_list` and `rf_max_wait_time_ms` in constructor of UnorderedInputStream.